src/framework/components/element/text-element.js

```diff
@@ -286,7 +286,7 @@ class TextElement {
         if (text === undefined) text = this._text;
 
         // get the list of symbols
-        this._symbols = string.getSymbols(text);
+        this._symbols = string.getSymbols(text.normalize('NFC'));
```

Comment:
Hey @slimbuck - `String#normalize` doesn't exist in IE11. 😭 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#browser_compatibility

Reply:
Addressed in https://github.com/playcanvas/engine/pull/2999